kube-webhook-certgen.yaml

```diff
@@ -0,0 +1,40 @@
+package:
+  name: kube-webhook-certgen
```

Comment:
I think this package is already a sub-package to ingress-nginx-controller. 

Ref: https://github.com/wolfi-dev/os/blob/fe966cd9bc3c757c952ac02f7750e5c01607e0b5/ingress-nginx-controller.yaml#L505-L513